x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -434,6 +434,16 @@ ROW a=["foo", "foo", "bar", "foo"]
 // end::mv_dedupe-result[]
 ;
 
+mvDedupeFromIndex
+from multivalues
+| eval b = mv_dedupe(boolean), k = mv_dedupe(keyword), i = mv_dedupe(integer), l = mv_dedupe(long), d = mv_dedupe(double)
+| keep b, k, i, l, d
+;
+
+b:boolean     | k:keyword      | i:integer | l:long | d:double
+[false, true] | ["bar", "foo"] | [1, 2]    | [1, 2] | [1.1, 2.2]
```

Comment:
I have no idea why, but without this fix, the multivalues are _not deduped_, but _indeed sorted_ when running the query from above; so this fails with

```
row 0 column 2:a list containing
row 0 column 2:0: expected "[1, 2]" but was "[1, 1, 2, 2]"
row 0 column 3:a list containing
row 0 column 3:0: expected "[1, 2]" but was "[1, 1, 2, 2]"
row 0 column 4:a list containing
row 0 column 4:0: expected "[1.1, 2.2]" but was "[1.1, 1.1, 2.2, 2.2]"
```

Reply:
It's because numeric *are* sorted, but not deduplicated. Disaster.